app/i2c.c

```
@@ -0,0 +1,348 @@
+            break;
+    }
+}
+
+/**
+ * @brief Initialize the master TWI.
+ *
+ * Function used to initialize the master TWI interface that would communicate with simulated EEPROM.
+ *
+ * @return NRF_SUCCESS or the reason of failure.
+ */
+int twi_master_init(void)
```

cppcheck (2.10-2)

[style] The function 'twi_master_init' is never used.

(unusedFunction)